NitroxClient/Communication/Exceptions/ParameterValidationException.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+
+namespace NitroxClient.Communication.Exceptions
+{
+    public class ParameterValidationException: Exception
+    {
+        public string FaultingParameterName { get; private set; }
```

Comment:
Since this property is only set in the constructor and will never be changed by the class, consider omitting `private set;`.